src/mmrelay/meshtastic_utils.py

```
@@ -1397,15 +1511,29 @@ def sendTextReply(
+# Backward-compatible alias for older call sites.
+sendTextReply = send_text_reply
 
 
 if __name__ == "__main__":
     # If running this standalone (normally the main.py does the loop), just try connecting and run forever.
     meshtastic_client = connect_meshtastic()
-    loop = asyncio.get_event_loop()
+    loop = asyncio.new_event_loop()
+    asyncio.set_event_loop(loop)
     event_loop = loop  # Set the event loop for use in callbacks
     loop.create_task(check_connection())
```

Ruff (0.14.11)

Store a reference to the return value of `loop.create_task`

(RUF006)